cookbook/recipes/common.rb

```diff
@@ -0,0 +1,19 @@
+node.set[:java][:jdk_version] = 7
+
+include_recipe 'java'
+include_recipe 'baragon::build'
+
+%w(/etc/baragon /var/log/baragon).each do |dir|
```

Comment:
If you changed one of those paths, in a Git diff, it would look like you changed the whole line. Perhaps it's personal style, but I like it when each item of an array has its own line, and changes to it are easily reflected in diffs. ¯\_(ツ)_/¯